src/index.js

```diff
@@ -23,15 +24,16 @@ window.addEventListener('DOMContentLoaded', () => {
 tasksListDisplay.addEventListener('click', (e) => {
   const target = e.target.closest('.fa-ellipsis-v');
   const trash = e.target.closest('.fa-trash-can');
+
   const listDisplay = document.querySelector('.display-list');
   if (target) {
     target.nextElementSibling.classList.toggle('show');
     target.style.display = 'none';
-    deleteTask(Number(target.id));
+    deletion.deleteTask(Number(target.id));
     localStorage.setItem('todo', JSON.stringify(listDisplay));
   }
 
   if (trash) {
-    deleteTask(Number(trash.id));
+    deletion.deleteTask(Number(trash.id));
   }
```

Comment:
Nice job so far👍 But you still haven't completely implemented the changes requested by the previous reviewer🥲 Your app still has some issues when calling the remove function🥲 Kindly take a look at the video below for clarity to see the current behavior of your app and make the necessary change👍



https://user-images.githubusercontent.com/64643995/228900081-4b81b271-7efb-4788-98f2-837da1e80f8b.mp4